utils/llm.py

```diff
@@ -124,7 +127,67 @@ def find_json(text):
         except Exception:
             return []
     else:
-        return [] 
+        return []
+
+
+def route_llm_prompt(text_chunk, instructions, llm_providers, source_langage, target_language):
+    """
+    Route the given prompt to the appropriate LLM model using Not Diamond.
+    """
+    nd_url = "https://not-diamond-server.onrender.com"
```

Comment:
This url is wrong. Should be "https://not-diamond-server.onrender.com/v2/TT/translate"

Reply:
Updated, thanks for catching.